app/services/store_sp_metadata_in_session.rb

```diff
@@ -24,20 +32,79 @@ def sp_request
     @sp_request ||= ServiceProviderRequestProxy.from_uuid(request_id)
   end
 
+  def ial_value
+    if parsed_vot.ialmax?
+      0
+    elsif parsed_vot.identity_proofing?
+      2
+    else
+      1
+    end
+  end
+
+  def ial2_value
+    parsed_vot.identity_proofing?
+  end
+
+  def ialmax_value
+    parsed_vot.ialmax?
+  end
+
+  def aal_level_requested_value
+    if parsed_vot.phishing_resistant?
+      3
+    elsif parsed_vot.aal2?
+      2
+    else
+      1
+    end
+  end
+
+  def piv_cac_requested_value
+    parsed_vot.hspd12?
+  end
+
+  def phishing_resistant_value
+    parsed_vot.phishing_resistant?
+  end
+
+  def biometric_comparison_required_value
+    parsed_vot.biometric_comparison?
```

Comment:
Here I think we want to look at `parsed_vot.biometric_comparison? || sp_request.biometric_comparison_required`. Since the current biometric comparison API is experimental and exists outside of the `acr_values` and `vtr` params we'll need to look at what is on the SP request until we update the sample OIDC SP.